chat-widget/src/components/livechatwidget/interfaces/ILiveChatWidgetProps.ts

```diff
@@ -70,4 +71,5 @@ export interface ILiveChatWidgetProps {
     initialCustomContext?: any;
     draggableChatWidgetProps?: IDraggableChatWidgetProps;
     mock?: IMockProps;
+    featureConfigProps: IFeatureConfigProps;
```

Comment:
Should `featureConfigProps` be nullable with `?` ? 

Based on what I see, if this props was not provided, the editor or linting may thrown an error that this props is not provided.